[new-product] Boundary (#7811)
captn3m0/endoflife.date: products/boundary.md

---
title: Hashicorp Boundary
category: server-app
tags: hashicorp

> iconSlug: boundary

Reviewer: ```suggestion
iconSlug: hashicorp
```

There is no `boundary` icon on https://simpleicons.org/

permalink: /boundary
versionCommand: boundary --version
releasePolicyLink: https://support.hashicorp.com/hc/en-us/articles/360021185113-Support-Period-and-End-of-Life-EOL-Policy

> changelogTemplate: https://github.com/hashicorp/boundary/blob/v__LATEST__/CHANGELOG.md

Reviewer: ```suggestion
changelogTemplate: https://github.com/hashicorp/boundary/blob/release/__RELEASE_CYCLE__.x/CHANGELOG.md
```

0.16.3 changelog is not visible on https://github.com/hashicorp/boundary/blob/v0.16.3/CHANGELOG.md.

identifiers:
-   repology: boundary

auto:
  methods:

>   -   git: https://github.com/hashicorp/boundary.git

Reviewer: ```suggestion
  -   git: https://github.com/hashicorp/boundary.git
      regex: '^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$'
```

0.x version are ignored by default.

# eol(x) = releaseDate(x+3)
releases:
-   releaseCycle: "0.19"
    releaseDate: 2025-02-10
    eol: false
    latest: '0.19.2'
    latestReleaseDate: 2025-05-08

-   releaseCycle: "0.18"
    releaseDate: 2024-10-14
    eol: false
    latest: '0.18.2'
    latestReleaseDate: 2024-12-12

-   releaseCycle: "0.17"
    releaseDate: 2024-07-31
    eol: false
    latest: '0.17.2'
    latestReleaseDate: 2024-10-14

-   releaseCycle: "0.16"
    releaseDate: 2024-04-30

>     eol: false

Reviewer: ```suggestion
    eol: 2025-02-10
```

`eol(x) = releaseDate(x+3)`

    latest: '0.16.3'
    latestReleaseDate: 2024-08-23

---

> > [Hashicorp Boundary](https://www.hashicorp.com/en/products/boundary) Standardize secure remote access

Reviewer: ```suggestion
> [Hashicorp Boundary](https://www.hashicorp.com/products/boundary) is an identity-aware proxy aimed at simplifying and securing least-privileged access to cloud infrastructure.
```

Based on https://developer.hashicorp.com/boundary/docs/overview/what-is-boundary.
